ansible-ipi-install/roles/installer/tasks/25_customize_filesystem.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Verify if {{ role_path }}/files/customize_filesystem/master/worker is empty
+  find:
+    paths: "{{ role_path }}/files/customize_filesystem/{{ item }}"
+  register: filesFound
+  with_items:
+    - "master"
+    - "worker"
+  delegate_to: localhost
```

Comment:
@rlopez133 This change isn't working correctly. I need to root-cause and fix it.

Reply:
@dustinblack what are you seeing?

Reply:
The find needed to recurse in order to return the files nested in the directories, and the when logic for the block needed to be a little stronger. Fixed now.